front/app/containers/UsersEditPage/FragmentForm.tsx

```diff
@@ -1,20 +0,0 @@
-import React, { useEffect, useState } from 'react';
-
-import Fragment from 'components/Fragment';
-
-import { getJwt } from 'utils/auth/jwt';
-
-export default () => {
-  const [jwt, setJwt] = useState<string | null>(null);
-
-  useEffect(() => {
-    setJwt(getJwt() || null);
-  }, []);
-
-  return (
-    <Fragment
-      name="profile-settings-section"
```

Comment:
The only fragment with this name in the AWS bucket directory `cl2-cms:fragments` is for http://oostende.stg.govocal.com/ which is a demo platform on staging.